mne/label.py

```diff
@@ -2666,3 +2669,49 @@ def select_sources(subject, label, location='center', extent=0.,
                           hemi=new_label.hemi, name=name, subject=subject)
 
     return new_label
+
+
+def find_label_in_annot(pos, subject='fsaverage', annot='aparc.a2005s+aseg',
```

Comment:
The default for `read_labels_from_annot` is `aparc` so this should be `aparc`. Also, nothing in this actually has to do with `mne.Label` objects, so let's rename
```suggestion
def find_pos_in_annot(pos, subject='fsaverage', annot='aparc+aseg',
```

Reply:
... I just realized what I think you meant by the original name, which is "Find the label in annot corresponding to a given pos". But I read it as "Find where a given mne.Label is in annot". Overall I think "find_pos_in_annot" being short for "Find where pos is in annot" makes the most sense here, especially since what you get back might be the name of a mne.Label (not a label object itself) or it might be the name of a region that you would use with a volumetric atlas.